src/app/features/profile/profile-detail/profile-detail.component.ts

```diff
@@ -15,13 +16,14 @@ export class ProfileDetailComponent implements OnInit {
 
     public profile$: Observable<IProfile>;
 
-    constructor (private store: Store<AppState>) {
+    constructor (private store: Store<AppState>, private notificationService: NotificationService) {
     }
 
     ngOnInit () {
 
         this.store.dispatch(getProfilePending());
         this.profile$ = this.store.select(getUserProfile);
+        this.notificationService.showNotification('Get Profile Success!', 3000, 'yellow');
```

Comment:
are you sure you will successfully get it here? you just tried to get it, it may fail and will be completed in some time